db/schema.rb

```diff
@@ -409,20 +425,20 @@
   add_index "translations", ["translatable_type", "translatable_id"], name: "index_translations_on_translatable_type_and_translatable_id", using: :btree
 
   create_table "users", force: :cascade do |t|
-    t.datetime "created_at", null: false
+    t.datetime "created_at",                            null: false
     t.datetime "current_sign_in_at"
     t.inet     "current_sign_in_ip"
-    t.string   "email", default: "", null: false
-    t.string   "encrypted_password", default: "", null: false
+    t.string   "email",                  default: "",   null: false
+    t.string   "encrypted_password",     default: "",   null: false
     t.datetime "last_sign_in_at"
     t.inet     "last_sign_in_ip"
-    t.boolean  "notify_on_new_logs", default: true
+    t.boolean  "notify_on_new_logs",     default: true
     t.integer  "profile_id"
     t.datetime "remember_created_at"
     t.datetime "reset_password_sent_at"
     t.string   "reset_password_token"
-    t.integer  "sign_in_count", default: 0, null: false
-    t.datetime "updated_at", null: false
+    t.integer  "sign_in_count",          default: 0,    null: false
+    t.datetime "updated_at",                            null: false
```

Comment:
I'm not sure how all these spaces got added in. We are using the fix-db-schema-conflicts gem which should prevent this. Can you try running your db:schema:dump again? Definitely don't want all these schema changes in the PR in any case please. Just what you're actually adding.